api/accounts.go

```diff
@@ -304,3 +313,30 @@ func (a *API) electionCountHandler(msg *bearerstdapi.BearerStandardAPIdata, ctx
 	}
 	return ctx.Send(data, bearerstdapi.HTTPstatusCodeOK)
 }
+
+// /accounts/<organizationID>/token/transfers/page/<page>
+// Returns the token transfers for an organization
+func (a *API) tokenTransfersHandler(msg *bearerstdapi.BearerStandardAPIdata, ctx *httprouter.HTTPContext) error {
+	organizationID, err := hex.DecodeString(util.TrimHex(ctx.URLParam("organizationID")))
+	if err != nil || organizationID == nil {
+		return fmt.Errorf("organizationID (%q) cannot be decoded", ctx.URLParam("organizationID"))
+	}
+	page, err := strconv.Atoi(ctx.URLParam("page"))
```

Comment:
I'd check if the account actually exists (by querying the state) before making the scrutinizer "expensive" call.

Reply:
oki, will do! thank you